resolvetosavelives/benchmarks
# Format Ruby files with Prettier on pre-commit

Closes ["automate Ruby formatting with Prettier"](https://www.pivotaltracker.com/story/show/174090065)

# What it does
* Upgrades Husky to  v5
* Installs the Prettier Ruby plugin
* Adds .rb, .erb, and .haml files to the list of file types formatted by the pre-commit hook.

# Testing
1. Run `yarn install` to ensure you have Husky v5. 
2. Edit a Ruby file to add unnecessary line breaks or whitespace at the end of a line.
3. Attempt to commit it.
4. You should see the pre-commit hook run. Git will complain that you're trying to make an empty commit because Prettier removed your bad formatting.

# Screenshots
<img width="820" alt="image" src="https://user-images.githubusercontent.com/200723/108439093-7a77b300-7205-11eb-946e-850b73250898.png">

# Stories
As a developer,
I want to format my Ruby files automatically with Prettier,
So my code style is consistent across the repo.
